src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -399,7 +389,7 @@
         [:<>
          background
          (when (or (not animated?) @render-camera?)
-           [render-camera show-camera? @qr-view-finder camera-ref on-read-code show-holes?])
```

Comment:
Removed the `show-holes?` check as it's not required now as it was used during the initial implementation of the sync screen where we wrapped the blur view with a hole view.   

Now, the background is passed as a prop (as it changes for the `sign-in` and `sync new device` screen) and it will contain the blur if needed.